README.md

```diff
@@ -100,6 +102,41 @@ python main_run_multitask.py --config-path [PATH] --config-name [CONFIG]
 ```
 Thanks to the modular nature of `hydra` you can reuse many of our config settings for your own experiments with Graphium.
 
+
+## First Time Running on IPUs
+For new IPU developers this section helps provide some more explanation on how to set up an environment to use Graphcore IPUs with Graphium. 
+
+```bash
+# Set up a virtual environment as normal
+mkdir ~/.venv                               # Create the folder for the environment
+python3 -m venv ~/.venv/graphium_ipu        # Create the environment
+source ~/.venv/graphium_ipu/bin/activate    # Activate the environment
+
+# We can download the Poplar SDK directly using `wget` - more details on the various Graphcore downloads can be found here `https://www.graphcore.ai/downloads`
+
+# NOTE: For simplicity this will download the SDK directly where you run this command, we recommend doing this outside the Graphium directory. 
```

Comment:
```suggestion
# NOTE: For simplicity this will download the SDK directly where you run this command, we recommend doing this outside the Graphium directory. 
# Make sure to download the right file according to your operating system
```